projects/mac_ai/testing/prepare_llama_cpp.py

```
@@ -55,8 +55,15 @@ def retrieve(prefix):
     retrieve("prepare.llama_cpp.release")
 
 
-def prepare_test(base_work_dir, platform):
-    retrieve_latest_version(base_work_dir)
+def prepare_test(base_work_dir, platform, cleanup=True):
+    try:
+        retrieve_latest_version(base_work_dir)
+    except Exception as e:
+        if not cleanup:
+            raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)